core/client/src/test/java/alluxio/hadoop/ConfUtilsTest.java

```diff
@@ -36,10 +34,10 @@ public void mergeEmptyHadoopConfigurationTest() {
     Configuration.defaultInit();
     org.apache.hadoop.conf.Configuration hadoopConfig = new org.apache.hadoop.conf.Configuration();
 
-    Map<String, String> before = Configuration.toMap();
+    long beforeSize = Configuration.toMap().size();
```

Comment:
The design of the existing tests only compared the size. If in need, I can add some more check for that test.